src/r_cmd.rs

```diff
@@ -50,6 +50,26 @@ pub fn find_r_version_command(r_version: &Version) -> Result<RCommandLine, Versi
         found_r_vers.push(path_r.original);
     }
 
+    // Include matching rig-formatted R on the path if it exists
+    // e.g. R-<major>.<minor>-<arch>
+    let info = os_info::get();
+    let major_minor = r_version.major_minor();
+    if let Some(arch) = info.architecture() {
```

Comment:
i think this should be only checked for on mac - windows needs to behave differently, including modifying the registry and pointing to different Rtools so won't work the same